diff --git a/node_modules/@vector-im/matrix-bot-sdk/.bun-tag-3de4bc31a68ff0a9 b/.bun-tag-3de4bc31a68ff0a9
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/lib/http.js b/lib/http.js
index dadaf8d7fc21b1578036ba820256709a136186c3..43584a0a3ce14077cbcd692a7a4845e611acc74f 100644
--- a/lib/http.js
+++ b/lib/http.js
@@ -67,7 +67,7 @@ async function doHttpRequest(baseUrl, method, endpoint, qs = null, body = null,
         LogService_1.LogService.error("MatrixHttpClient", "(REQ-" + requestId + ")", e);
         throw e;
     });
-    let resBody = Buffer.from(await response.body.bytes());
+    let resBody = Buffer.from(await response.body.arrayBuffer());
     if (!noEncoding) {
         resBody = resBody.toString();
         try {
